lib/i18n/index.ts

```diff
@@ -0,0 +1,66 @@
+import { po } from 'gettext-parser';
+import Gettext from 'node-gettext';
+import { logger } from '../logger';
+import { readSystemFile } from '../util/fs';
+
+const gt = new Gettext();
+
+export function _(msgid: string): string {
+  return gt.gettext(msgid);
+}
+
+export function ngettext(
```

Comment:
Is it going too far to suggest something like `__()` for the name of this function, given we exported `gettext` as `_()`?

Reply:
Hi @JamieMagee, if my understanding is correct, you'd suggest rename the method `ngettext()` to `__()`?

We used the  method `_()`, because it is listed in the `Gettext` manual as a formal 'keyword' for JavaScript language:

> For JavaScript: _, gettext, dgettext:2, dcgettext:2, ngettext:1,2, dngettext:2,3, pgettext:1c,2, dpgettext:2c,3.

It will not surprise a developer for its wired name, after they read the Gettext [manual](https://www.gnu.org/software/gettext/manual/gettext.html#Choice-of-input-file-language).